test/e2e/data/e2e_conf.yaml

```diff
@@ -12,11 +12,11 @@
 managementClusterName: capo-e2e
 
 images:
-- name: gcr.io/k8s-staging-cluster-api/cluster-api-controller:v1.0.0
+- name: gcr.io/k8s-staging-cluster-api/cluster-api-controller:v1.0.4
```

Comment:
Shouldn't be the cause of the issue but those versions should also be updated in the makefile in the `test-e2e-image-prerequisites` target